### PR TITLE
purescript 0.6.3

### PR DESCRIPTION
**Pull request by jstepien**
PureScript is a small strongly typed programming language that compiles to JavaScript.

See also:

  - http://www.reddit.com/r/purescript
  - http://www.purescript.org/
  - http://try.purescript.org/
  - https://github.com/purescript/purescript

### CODE REVIEW

File: Library/Formula/purescript.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+       - https://ghc.haskell.org/trac/ghc/ticket/9711
+    EOS
+  end
+
+  def install
+    cabal_sandbox do
+      cabal_install "--only-dependencies"
+      cabal_install "--prefix=#{prefix}"
+    end
+    cabal_clean_lib
```
Comment (jacknagel) [inline Library/Formula/purescript.rb]: I think the body of this method is the same as just calling `install_cabal_package`?

File: Library/Formula/purescript.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require "formula"
```
Comment (jacknagel) [inline Library/Formula/purescript.rb]: This line can be removed.

File: Library/Formula/purescript.rb (hunk 1 above)
Comment (jstepien) [inline Library/Formula/purescript.rb]: Thanks for the comment; fixed in 3e6b845.

File: Library/Formula/purescript.rb (hunk 2 above)
Comment (jstepien) [inline Library/Formula/purescript.rb]: Removed in 3c548a84

File: Library/Formula/purescript.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  depends_on "cabal-install" => :build
+
+  fails_with :clang do
+    cause <<-EOS
+      GHC with clang fails to compile text-1.2.0.0. See also:
+
+       - http://git.io/L5a_JA
+       - https://ghc.haskell.org/trac/ghc/ticket/9711
+    EOS
+  end
```
Comment (mikemcquaid) [inline Library/Formula/purescript.rb]: Would like to see an upstream patch before we merge this. Clang is the default compiler on OSX so things that don't work with Clang arguably aren't ported to OSX.

**Comment by jacknagel** [comment]
A different PR was ultimately pulled.

**Pull request closed without merging**